### ISSUE TITLE
[SUGGESTION] [BUILDING] Addition of two lore-important maps.

### ISSUE THREAD

**Issue by fynnconstantine** [DrewBokman/AOTG-Public-Issue-Board#78]
**Is your feature request related to a problem? Please describe.**

Somewhat. There are two issues that could be solved here, the lack of possibilities for lore and the lack of interaction with Government and society.

**Describe the solution you'd like**

The addition of two new maps: Prison and Royal Palace.
Palace would be the most important of the two. It would basically be a place for the nobility to host ceremonies, speeches, and parties, as well as host a Throne room. The reason why this is a good idea is because it creates lots of opportunities for lore, one example being a party with the nobility, military HICOMM and wealthy citizens, where one person is poisoned or kidnapped during the event, starting a mission or having other things unfold.

Prison is also a great idea. It would be, in concept, a prison facility in a snowy, remote place, where the convicted would pay for their crimes with incarceration, forced labor, and harsh living conditions. People who were convicted in Trial or are being held under MP custody could be sent to this place (and temporarily blacklisted from other maps, depending on the length of their sentence). The Royal Government would administrate the facility and produce documents, and the MP would keep watch of the facility during missions.

TL;DR Adding a royal palace for lore flavor and creativity and adding a prison to keep the banned (sentenced) players inside, giving the gov stuff to do and adding lore opportunities too.

It would be very much appreciated. Thank you.
